Codespaces: Building container failed

Hey hey!! 

I am trying to run encore on Codespaces and it seems like something goes wrong in the `.devcontainer`. The codespace fails to build and automatically runs into recovery mode. These are the error creation logs:

```
  > [6/6] RUN bash /scripts/godeps.sh:
 #11 12.82 go: downloading github.com/rivo/uniseg v0.2.0
 #11 12.89 go: downloading github.com/shurcooL/sanitized_anchor_name v1.0.0
 #11 19.64 + go install honnef.co/go/tools/cmd/staticcheck@latest
 #11 19.85 go: downloading honnef.co/go/tools v0.3.3
 #11 20.09 go: downloading golang.org/x/exp/typeparams v0.0.0-20220218215828-6cf2b201936e
 #11 20.09 go: downloading golang.org/x/tools v0.1.11-0.20220513221640-090b14e8501f
 #11 20.57 go: downloading github.com/BurntSushi/toml v0.4.1
 #11 20.58 go: downloading golang.org/x/sys v0.0.0-20211019181941-9d821ace8654
 #11 20.58 go: downloading golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
 #11 20.93 pkg/mod/honnef.co/go/tools@v0.3.3/go/ir/builder.go:36:2: //go:build comment without // +build comment
 ------
 ERROR: failed to solve: executor failed running [/bin/sh -c bash /scripts/godeps.sh]: exit code: 1
 Stop (60083 ms): Run: docker buildx build --load --build-arg BUILDKIT_INLINE_CACHE=1 -f /var/lib/docker/codespacemount/workspace/encore/.devcontainer/Dockerfile -t vsc-encore-c80451401b0e9c470aa7d7418c3f6b29 /var/lib/docker/codespacemount/workspace/encore/.devcontainer
 Error: Command failed: docker buildx build --load --build-arg BUILDKIT_INLINE_CACHE=1 -f /var/lib/docker/codespacemount/workspace/encore/.devcontainer/Dockerfile -t vsc-encore-c80451401b0e9c470aa7d7418c3f6b29 /var/lib/docker/codespacemount/workspace/encore/.devcontainer
     at OF (/usr/lib/node_modules/@microsoft/vscode-dev-containers-cli/dist/spec-node/devContainersSpecCLI.js:258:1490)
     at processTicksAndRejections (internal/process/task_queues.js:95:5)
     at async Wf (/usr/lib/node_modules/@microsoft/vscode-dev-containers-cli/dist/spec-node/devContainersSpecCLI.js:257:2801)
     at async pb (/usr/lib/node_modules/@microsoft/vscode-dev-containers-cli/dist/spec-node/devContainersSpecCLI.js:257:1779)
     at async DF (/usr/lib/node_modules/@microsoft/vscode-dev-containers-cli/dist/spec-node/devContainersSpecCLI.js:263:2006)
     at async to (/usr/lib/node_modules/@microsoft/vscode-dev-containers-cli/dist/spec-node/devContainersSpecCLI.js:263:3110)
     at async Ak (/usr/lib/node_modules/@microsoft/vscode-dev-containers-cli/dist/spec-node/devContainersSpecCLI.js:383:8108)
     at async Ok (/usr/lib/node_modules/@microsoft/vscode-dev-containers-cli/dist/spec-node/devContainersSpecCLI.js:383:7864)
 Stop (60326 ms): Run: /usr/bin/node /usr/lib/node_modules/@microsoft/vscode-dev-containers-cli/dist/spec-node/devContainersSpecCLI.js up --user-data-folder /var/lib/docker/codespacemount/.persistedshare --container-data-folder .vscode-remote/data/Machine --container-system-data-folder /var/vscode-remote --workspace-folder /var/lib/docker/codespacemount/workspace/encore --id-label Type=codespaces --log-level info --log-format json --config /var/lib/docker/codespacemount/workspace/encore/.devcontainer/devcontainer.json --override-config /root/.codespaces/shared/merged_devcontainer.json --default-user-env-probe loginInteractiveShell --remove-existing-container --mount type=bind,source=/.codespaces/agent/mount/cache,target=/vscode --skip-post-create --update-remote-user-uid-default never --mount-workspace-git-root false
 Exit code 1
 Failed to create container.
 Error: Command failed: /usr/bin/node /usr/lib/node_modules/@microsoft/vscode-dev-containers-cli/dist/spec-node/devContainersSpecCLI.js up --user-data-folder /var/lib/docker/codespacemount/.persistedshare --container-data-folder .vscode-remote/data/Machine --container-system-data-folder /var/vscode-remote --workspace-folder /var/lib/docker/codespacemount/workspace/encore --id-label Type=codespaces --log-level info --log-format json --config /var/lib/docker/codespacemount/workspace/encore/.devcontainer/devcontainer.json --override-config /root/.codespaces/shared/merged_devcontainer.json --default-user-env-probe loginInteractiveShell --remove-existing-container --mount type=bind,source=/.codespaces/agent/mount/cache,target=/vscode --skip-post-create --update-remote-user-uid-default never --mount-workspace-git-root false
 Error Code: 1302
 Container creation failed.
```

Is there something I can do about the config to fix this?

--- a/.devcontainer/Dockerfile
+++ b/.devcontainer/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.16
+FROM golang:1.19
 
 RUN apt-get update && apt-get install -y sudo
 RUN curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash - && \

--- a/.devcontainer/scripts/install.sh
+++ b/.devcontainer/scripts/install.sh
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 set -e
 
-encore_uri=$(curl -sSf -N "https://encore.dev/api/releases?target=linux_amd64&show=url")
+target="$(go env GOOS)_$(go env GOARCH)"
+
+encore_uri=$(curl -sSf -N "https://encore.dev/api/releases?target=${target}&show=url")
 if [ ! "$encore_uri" ]; then
     echo "Error: Unable to determine latest Encore release." 1>&2
     exit 1

